src/System.Linq.Expressions/src/System.Linq.Expressions.builds

```diff
@@ -4,9 +4,6 @@
   <ItemGroup>
     <Project Include="System.Linq.Expressions.csproj" />
     <Project Include="System.Linq.Expressions.csproj">
```

Comment:
I might come to that later. For now I'm happy to have this one ticking green. There's no better code than code you'd previously concluded wasn't possible 😄